unit_tests/test_nemesis_sisyphus.py

```diff
@@ -71,7 +71,7 @@ def test_list_all_available_nemesis(generate_file=True):
     disruption_list, disruptions_dict, disruption_classes = sisyphus.get_list_of_disrupt_methods(
         subclasses_list=subclasses, export_properties=True)
 
-    assert len(disruption_list) == 81
+    assert len(disruption_list) == 82
```

Comment:
also the nemesis.yml should be updated.
I don't understand how the checks are passing without it.

Reply:
I think there is a bug in the test and the generate_file should be set to false by default.
we probably merged it with the wrong version of it.
Anyway, you should include your nemesis.yml and nemesis_classes.yml files here.

Reply:
added